NEWS

```diff
@@ -13,8 +13,14 @@
 
 ## 1.16.0-rc4 (February 21, 2024)
 ### Bugfixes:
+#### UCP
```

Comment:
can we also add details for #9723?

Reply:
isn't it covered by the comment for #9705 ?

Reply:
Maybe rephrase it to "Fixed mlx5 WQE posting error due to compiler memory copy optimizations"
and rename the section from "UCT" to "RDMA CORE" like in line 56